tests/FamilyHubs.ServiceDirectory.Api.FunctionalTests/BaseWhenUsingApiUnitTests.cs

```diff
@@ -123,33 +123,6 @@ private HttpRequestMessage CreateHttpRequestMessage(HttpMethod verb, string path
 
         return request;
     }
-
-    protected bool IsRunningLocally()
```

Comment:
If there's a reason we're excluding the functional tests in CI (I guess because they run before deployment?) we should just filter them out rather than adding this complex logic.

`dotnet test --filter 'FullyQualifiedName!~FunctionalTests'`

Reply:
Turns out they run fine 😎